static/js/message_edit.js

```diff
@@ -819,8 +819,8 @@ export function save_message_row_edit(row) {
 
                     row = message_lists.current.get_row(message_id);
                     if (!is_editing(message_id)) {
-                        // Return to the message editing open UI state.
-                        start_edit_maintaining_scroll(row, echo_data.orig_raw_content);
+                        // Return to the message editing open UI state with the edited content.
+                        start_edit_maintaining_scroll(row, echo_data.raw_content);
```

Comment:
This seems to have been present since the first message_edit local echo PR: https://github.com/zulip/zulip/pull/12145

Can you scan the history of that PR for whether this decision was discussed?

Reply:
https://github.com/zulip/zulip/pull/12145#pullrequestreview-228998391 suggests that the PR originally the new content here (which presumably got mis-refactored at some point to `orig_raw_content`).

Reply:
Yes, so I am sure this change is correct.